src/Loader.php

```diff
@@ -160,6 +160,7 @@ public static function add_blocks(): void
         new Blocks\Accordion();//NOSONAR
         new Blocks\Cookies();//NOSONAR
         new Blocks\Counter();//NOSONAR
+        Blocks\PostsList::registerHooks();
```

Comment:
Shouldn't we have this code run after the Beta Blocks check, since it affects the Posts List block only?

Reply:
Good point, fixed :+1: